### DISCUSSION TITLE
All Features - It is not functional

### DISCUSSION THREAD

**Discussion by luizlinxsys** [chatwoot/chatwoot#6299] [Chatwoot APIs]
Hey guys, I would like to know if you are also having problems with the operation of this function, in some versions it does not work the check, in others it is possible to change from true to false, but in the system it has no effect. Continue in the same way.

All help is welcome =)

Success to all!

![image](https://user-images.githubusercontent.com/12704468/213443729-15fcba99-6873-4c6e-a8b6-0c1dd795c08e.png)

**Comment by cahflitto** [comment]
Hi man, I see it the same. There are functions that work and some that don't work, even I changed them forcefully through postgres and already correctly I reset the redis but there is no change.
In the previous version I changed it by force through postgres and it had its effect.

For the latest version, I changed it through the UI directly instead of there are features that I don't show.

![false](https://user-images.githubusercontent.com/98459570/213860219-2396841c-1384-4683-9766-2ed53606c77d.png)

We hope that in the future some of these functions can run better, everything is already very good and even almost perfect thank you.
